index.html

```diff
@@ -0,0 +1,154 @@
+          </span>
+        </a>
+        <div class="close" id="cancel-btn">x</div>
+        <ul class="nav-menu">
+          <li> <a href="about.html">About</a></li>
+          <li> <a href="#">Program</a></li>
+          <li> <a href="#">News</a></li>
+          <li> <a href="#">Sponsor</a></li>
+          <li> <a class="campaign-item" href="#" id="button">CC Campaign</a></li>
+        </ul>
+      </div>
+
+
+    </div>
+    <!-- <div class="logo"><img src="./images/Restaurant-logo.jpg" alt=""></div> -->
+  </nav>
+  <header>
+    <h3 class="open-comments">HEY YOU THERE COME TASTE SOME!</h3>
+    <h1 class="welcome">WELCOME TO CHEF YOUNG'S RESTAURANT AKA THE MOST FAMOUS AND SUMPTUOUS RESTAURANT IN THE WORLD
+      OF 2023</h1>
+    <div class="paragraph">
+      <p>It is often said "The road to a man's heart is through his stomach", We warmly welcome you to our beautiful
+        and inviting restaurant, where we offer satisfying and quality food. A very spacious place for your lovely
+        Dates. </p>
+    </div>
+    <div class="location">
+      <h1>2023.12.01(FRI) ~ 31(SAT)</h1>
+      <p>@Gaius street near the Fancy park, Douala, Cameroon</p>
+    </div>
+  </header>
+  </div>
+
+  <section class="vicinity">
+    <h3 class="h3-vicinity">Vicinity</h3>
+    <div class="line"></div>
+
+    <div class="cards">
+      <div class="main-container">
+        <img class="image images" src="images/image-icon.jpg" alt="front-view">
+        <h4 class="heading">WELcome</h4>
+        <P class="welcome-p">Here we have the front view of our restaurant and as you can see </P> 
+      </div>
+
+      <div class="main-container">
+        <img class="image images" src="images/image-icon.jpg" alt="reception-area">
+        <h4 class="heading">WELcome</h4>
+        <P class="welcome-p">Here we have the front view of our restaurant and as you can see </P> 
+      </div>
+
+      <div class="main-container">
+        <img class="image images" src="images/image-icon.jpg" alt="Romantic-date">
+        <h4 class="heading">WELcome</h4>
+        <P class="welcome-p">Here we have the front view of our restaurant and as you can see </P> 
+      </div>
+
+      <div class="main-container">
+        <img class="image images" src="images/image-icon.jpg" alt="Family-site">
+        <h4 class="heading">WELcome</h4>
+        <P class="welcome-p">Here we have the front view of our restaurant and as you can see </P> 
+      </div>
+
+      <div class="main-container">
+        <img class="image images" src="images/image-icon.jpg" alt="">
+        <h4 class="heading">WELcome</h4>
+        <P class="welcome-p">Here we have the front view of our restaurant and as you can see </P> 
+      </div>
+
+    </div class="button">
+    <button class="btn">PLACE YOUR ORDER</button>
+   <!-- <div class="see-t"><p class="see">SEE THE WHOLE PROGRAM</p></div>  -->
+  </section>
+
+  <section>
+    <h2 class="menu-list">MENU LIST</h2>
+    <div class="line"></div>
```

Comment:
- [ ] **[OPTIONAL]** It would be fantastic if you could consider adding images of the same size in the partners section. Currently, they don't appear aligned. Adding images with consistent dimensions will greatly enhance the visual appeal. 

   ### Your design:
   ![image](https://github.com/Young-Gaius/capstone-project/assets/107952724/7aab98d5-3596-427b-bb07-0dd136b6bba4)